apps/ui/modules/reservationUnit.ts

```diff
@@ -223,54 +229,56 @@ export const getFuturePricing = (
     : futurePricings[0];
 };
 
+function formatPrice(
+  price: number,
+  trailingZeros: boolean,
+  toCurrency?: boolean
+) {
+  const currencyFormatter = trailingZeros ? "currencyWithDecimals" : "currency";
+  const floatFormatter = trailingZeros ? "twoDecimal" : "strippedDecimal";
+  const formatters = getFormatters("fi");
+  const formatter = formatters[toCurrency ? currencyFormatter : floatFormatter];
+  return parseFloat(price.toString()) ? formatter.format(price) : 0;
```

Comment:
????

Why covert `number` -> `string` -> `number`?
Why return `string | number` union?

Reply:
I have no idea - this is code that was already there. I might have relocated it but that whole `formatter` thing was already there (and seemed to work)..

Reply:
ok, it's awful, lets not touch it